From 8ac8f30f921ac0cd66776e671f1c48621dfde249 Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Tue, 15 Mar 2022 15:33:02 -0300
Subject: [PATCH 112/163] math: Fix clang warnings for math/test-tgmath-ret.c

clang warns that since the global variables are only used to function
calls (without being actually used), there are not needed and will
not be emitted.
---
 math/test-tgmath-ret.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/math/test-tgmath-ret.c b/math/test-tgmath-ret.c
index 8c1cce37bd..acfb6e66e8 100644
--- a/math/test-tgmath-ret.c
+++ b/math/test-tgmath-ret.c
@@ -21,13 +21,20 @@
 #include <tgmath.h>
 #include <stdint.h>
 #include <stdio.h>
+#include <libc-diag.h>
 
+/* clang warns that since the global variables are only used to function
+   calls (without being actually used), there are not needed and will not
+   be emitted.  */
+DIAG_PUSH_NEEDS_COMMENT_CLANG;
+DIAG_IGNORE_NEEDS_COMMENT_CLANG (13, "-Wunneeded-internal-declaration");
 static float fx;
 static double dx;
 static long double lx;
 static int rm = FP_INT_UPWARD;
 static unsigned int width = 64;
 static int errors = 0;
+DIAG_POP_NEEDS_COMMENT_CLANG;
 
 static void
 our_error (const char *c)
-- 
2.37.2

